src/settings.ts

```diff
@@ -59,6 +59,8 @@ export const VENUE_NAME_MIN_CHAR_COUNT = 3;
 export const VENUE_NAME_MAX_CHAR_COUNT = 50;
 export const PLAYA_VENUE_NAME = "Jam";
 export const PLAYA_VENUE_ID = "jamonline";
+export const GIFT_TICKET_MODAL_URL =
+  "https://here.burningman.org/event/virtualburn";
```

Comment:
[note] This is the URL for the 2021 online burn, taken from the link on http://sparklever.se/

Reply:
I wonder, why to keep any specific event vendoring in the main/staging version and not just replace with "" so too be customized for a given event?

Reply:
Minimising the scope of changes for this PR. The better long term fix would be to load this from configure the database, but that’s OOS here.

Reply:
+1 on centralizing event specific configuration (later) and minimizing the score of the pr. 
What will need to be done for ohbm (in env/ohbm ?) to not trigger any eventbrite related functionality/calls?

Reply:
Nothing, this PR removes everything eventbrite/related except for this static string.